parsing/extensions_parsing.mli

```diff
@@ -144,6 +145,14 @@ module type Of_ast_parameters = sig
       [[%extensions.comprehensions]] node, and the argument to that
       node is passed in as the [Parsetree] AST.
 
+      So, for example, if [of_ast] spots the expression
+
+      {[
+        [%extensions.comprehensions] blah
+      ]}
+
+      then it will call [of_ast_internal Comprehensions blah].
+
```

Comment:
I wonder if we should revise this whole section to be less specific about the representation?  But it's probably fine.